packages/bits/src/lib/select-v2/base-select-v2.ts

```diff
@@ -138,6 +143,7 @@ export abstract class BaseSelectV2 implements AfterViewInit, AfterContentInit, C
     private _selectedOptions: SelectV2OptionComponent[] = [];
 
     private _ariaLabel: string = "";
+    private _virtualScrollResizeObserver: ResizeObserver;
```

Comment:
By convention, Nova typically uses leading underscores only for private members with a corresponding public getter/setter.

```suggestion
    private virtualScrollResizeObserver: ResizeObserver;
```

Reply:
renamed it, thanks

Reply:
Added it [here](https://github.com/solarwinds/nova/pull/276/files#diff-d2adc1eebb3bcd5bf6298ad22b60cccfee0e21315462af82bbdd40aa4b073f57)